__tests__/integration.test.js

```diff
@@ -90,6 +94,8 @@ describe("#3 Tests route: POST /users/newuser", ()=>{
                           .post('/users/newuser')
                           .send(newUser)
                           .set('Content-Type', 'application/json');
+    //console.log(response.body);
+
     expect(response.statusCode).toBe(201);
   });
   test("#3.4 Should return 400", async ()=>{
```

Comment:
Ficaram muito escritos seus testes de integração, manolo, parabéns!

Uma melhoria que eu vejo:

Pra descrever melhor oq seu teste testa, e o que seu código faz, vc poderia explicar melhor o cenário. Isso é:
```suggestion
  test("#3.4 Should return 400, when invalid input is provided", async ()=>{
```